modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPRequest.macro

```diff
@@ -115,4 +121,18 @@ definition {
 		MenuItem.click(menuItem = ${key_option});
 	}
 
+	macro viewOptionsPresentModal {
+		for (var option : list ${option}) {
+			AssertElementPresent(
+				key_text = ${option},
+				locator1 = "Button#ANY");
+		}
+	}
+
+	macro viewPresentCommentBox {
+		AssertElementPresent(
+			key_fieldId = "messageDescribed",
+			locator1 = "TextArea#ANY_ID");
+	}
+
```

Comment:
Hey @joelrocha68 , I think theses macros are not necessary, due to the flows that clicks on these buttons and  type in the comment box field already assert that this elements are visible and working.